stoplight.js

```diff
@@ -0,0 +1,13 @@
+console.log("hi");
+
+// var button = $("#stoplight");
+// function handlClickEvent() {
+//   console.log("I was Clicked")
+// }
+document.getElementById('red').onclick = illuminateRed;
+document.getElementById('yellow').onclick = illuminateYellow;
+document.getElementById('green').onclick = illuminateGreen;
+
+function illuminateRed() {
+  document.getElementById('red').style.backgroundColor = 'red';}
+}
```

Comment:
I would have made use of jQuery's `.addClass` method and the `active` class in the stylesheet.